src/scalper.ts

```diff
@@ -925,14 +959,57 @@ export class Scalper {
       selfTradeBehavior: 'abortTransaction',
     });
     gammaOrders.add(gammaAskTx.transaction);
+    
+    // Calculate effective gamma subject to max at price
+    const whaleBidDiff = fairValue - whaleBidPrice;
+    const whaleAskDiff = whaleAskPrice - fairValue;
+    const backBidPrice = whaleBidDiff > fairValue - gammaBid ? 
+      Math.floor((whaleBidPrice + this.tickSize) * (1 / this.tickSize)) / (1 / this.tickSize) : undefined;
+    const backAskPrice = whaleAskDiff > gammaAsk - fairValue ? 
+      Math.floor((whaleAskPrice - this.tickSize) * (1 / this.tickSize)) / (1 / this.tickSize) : undefined;
+    const whaleBidGammaQty = Math.min(netGamma * (maxBackGammaMultiple - 1), dipTotalGamma * whaleBidDiff - netGamma);
```

Comment:
add comments to explain these calculations or make the code speak for itself

Reply:
L963

Reply:
That doesnt really explain what is going on everywhere. Like what is
(1 / this.tickSize)) / (1 / this.tickSize)

that shows up in a few places

Reply:
rounding off to the min tick size for order entry, will add